skins/cat17/address-form/src/__tests__/store.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { getters } from '../store'
+import {AddressState, Validity} from '../types';
+
+function newMinimalStore( overrides: Object ): AddressState {
+	return Object.assign(
+		{
+			isValidating: false,
+			form: {
+				salutation: Validity.INCOMPLETE,
+				title: Validity.INCOMPLETE,
+				firstName: Validity.INCOMPLETE,
+				lastName: Validity.INCOMPLETE,
+				companyName: Validity.INCOMPLETE,
+				street: Validity.INCOMPLETE,
+				postcode: Validity.INCOMPLETE,
+				city: Validity.INCOMPLETE,
+				country: Validity.VALID,
+				addressType: Validity.VALID
+			}
+		},
+		overrides
+	);
+}
+
+describe('Store', () => {
+
+	it('returns no invalid fields on render', () => {
+		expect(getters.invalidFields( newMinimalStore( {} ), null, null, null ) ).toStrictEqual( [] );
```

Comment:
You probably can't get rid of those `null` values, right?

Reply:
Nope, because Vuex's `Getter` type expects `(state: S, getters: any, rootState: R, rootGetters: any)`